pkg/gator/test/test_test.go

```diff
@@ -127,6 +129,24 @@ func TestTest(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "referential constraint with violation, referential data disabled",
+			inputs: []string{
+				fixtures.TemplateReferential,
+				fixtures.ConstraintReferential,
+				fixtures.ObjectReferentialInventory,
+				fixtures.ObjectReferentialDeny,
+			},
+			optionMutators: []TestOptionMutator{DisableReferentialData},
+			err:            constraintclienterrors.ErrInvalidConstraintTemplate,
+			// QUESTION FOR MAX: This really should be the below error.
+			// However, the errors.Is() check isn't working for the commented error.  I traced the
+			// error through the constraint framework and back to here and it
+			// appears to be using the %w directive at each level, which is
+			// what makes the wrapping work (AFAICT).  Perhaps you can see what
+			// I'm doing wrong
+			// err:            regorewriter.ErrDataReferences,
```

Comment:
There is nothing obvious. Can you print the contents of the error returned so we have concrete data to work with?